app/components/header.tsx

```diff
@@ -26,65 +26,73 @@ export default function Header() {
     <Flex
       as="header"
       backgroundColor="brand.500"
-      height="auto"
       width="100%"
       alignItems="center"
       shadow="2xl"
       fontSize="18"
       paddingY="2"
-      paddingX={{ base: '1', md: '10' }}
+      paddingX={{ base: '1', md: '5' }}
       justifyContent="space-between"
+      position="absolute"
```

Comment:
Position absolute causes empty space on the domain creation page:
<img width="955" alt="image" src="https://user-images.githubusercontent.com/67077705/224175942-40d192bc-d9ed-4447-8979-fdf16143705d.png">
Maybe something else can be used to get the same result?

Reply:
Could you please give an example of the overflow you fixed by setting `position: absolute` ?

Reply:
That's actually due to adding a marginTop on the container which is the header is located on the route. By adding the absolute it basically makes the header as part of the container instead of being outside of it.

Reply:
Sorry, could you please show which marginTop you are talking about here?

Reply:
![image](https://user-images.githubusercontent.com/32577022/224177974-a42bfa4a-4148-437d-92ad-bfcc170909d0.png)

Add a paddingY on the container, and that should solve the issue

Reply:
Removing margin top hides the title (`Create new DNS Record`):
<img width="700" alt="image" src="https://user-images.githubusercontent.com/67077705/224178003-ddd17aa1-fc2f-446b-98ec-2fe219e80fc8.png">

I believe this is because `position: absolute` makes it so that the header is no longer considered when rendering other elements. Essentially, it allows the header to overlap other elements. For example, on the domains table page:
With `position: absolute`:
<img width="701" alt="image" src="https://user-images.githubusercontent.com/67077705/224178453-f6ae3a97-6efe-42d2-bcca-f1b50ae18aec.png">

Without `position: absolute`:
<img width="702" alt="image" src="https://user-images.githubusercontent.com/67077705/224178399-e03f81e4-8465-4c16-8dba-c860a021f62b.png">

The content is pushed up because the header no longer takes that space when the page is rendered

Reply:
Is there a use case for `height: 100%` on the containers? Shouldn't `height: auto` work as well?

Reply:
I guess currently we don't really have any use of setting height to 100%, sure I'll remove it